opengemini/write.go

```diff
@@ -105,7 +165,38 @@ func (c *client) WritePoint(ctx context.Context, database string, point *Point,
 	return nil
 }
 
-func (c *client) internalBatchSend(ctx context.Context, database string, resource <-chan *sendBatchWithCB) {
+func (c *client) WriteBatchPointsWithRp(database string, rp string, bp []*Point) error {
```

Comment:
duplicate code, the same with `WriteBatchPoints` , try to make it simpler

Reply:
accept